src/components/CityColumnChart/ChartTable/ChartTable.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import numeral from 'numeral';
+import { INDICATORS } from '../../../constants';
+
+const ChartTable = props => (
+  <table className={props.className}>
+    <caption>
+      Table showing {props.chart.name} for each city. {props.chart.description}
+    </caption>
+
+    <thead>
+      <tr>
+        <th scope="col">City</th>
```

Comment:
The `scope` attribute added as suggested here: https://webaim.org/techniques/tables/data